modules/mod_authentication/templates/logon_error/message.reminder.tpl

```diff
@@ -0,0 +1,4 @@
+    <p>{_ You've entered an unknown username or email address. Please try again. _}</p>
```

Comment:
Do we still show this message after #1910?

Reply:
Actually not, the error is now a nondescript `pw`, so let's remove this message.